corehq/apps/accounting/utils.py

```diff
@@ -278,3 +278,41 @@ def get_default_domain_url(domain):
         DefaultProjectSettingsView.urlname,
         args=[domain],
     )
+
+
+def ensure_grant(grantee_slug, priv_slug, dry_run=False, verbose=False):
+    """
+    Adds a parameterless grant between grantee and priv, looked up by slug.
+    """
+
+    if dry_run:
+        grants = Grant.objects.filter(from_role__slug=grantee_slug,
+                                      to_role__slug=priv_slug)
+        if not grants:
+            logger.info('[DRY RUN] Granting privilege: %s => %s', grantee_slug, priv_slug)
```

Comment:
Noticed you just copied this, so no action required, but do you think it's worthwhile to add "privilege already granted" to the dry run? might be useful for auditing if we accidentally granted a privilege to a role that shouldn't be there